### PR TITLE
The great I ded blood cult melee combat nerf pr

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑
tweak: blood cult great sword damage reduced to 25
tweak: blood cult throw spear damage reduced to 30
tweak: you now need to use 20 force to break the mirror shield
tweak: mirror shield now shatters on hitting a non cult!
/🆑

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
this sucks

**Comment by HomocidlePancakes** [comment]
Bastard sword is unabtainable. Why are you nerfing it nerd?

**Comment by Takahiru** [comment]
mirror shield... shatters.... on one hit.. please no

**Comment by fluffe9911** [comment]
@HomocidlePancakes better safe then sorry nerd
@Takahiru Its always shattered on one hit it was just almost impossible to destroy cause you had to do 30 damage to it in a single hit which the only thing that could really do that is shotgun bullets

**Comment by SomeguyManperson** [comment]
Very few readily available guns deal 30 or more damage, mainly shotgun slugs, 357 rounds (require the detgun to be rigged and has a chance to backfire every shot), and 10mm rounds (only chambered by stechkins which have to be found on lavaland/in space).

**Comment by ShadowDeath6** [comment]
all of these are overtweaked, add 5 damage back to the weapons and make the speed robes 1.5x rather than 2x so people aren’t being 2 shot by easily spammable projectiles

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -994,8 +994,10 @@
 			if(!L.anti_magic_check())
 				if(is_servant_of_ratvar(L))
 					L.Paralyze(60)
+					break_mirror(T) //yogs added a break mirror proc
 				else
 					L.Paralyze(30)
+					break_mirror(T) //yogs added a break mirror proc
```
Comment (ShadowDeath6) [inline code/modules/antagonists/cult/cult_items.dm:999]: Why?

File: code/modules/antagonists/cult/cult_items.dm [hunk 2]
```diff
@@ -931,7 +931,7 @@
 		if(istype(hitby, /obj/item/projectile))
 			var/obj/item/projectile/P = hitby
 			if(P.damage_type == BRUTE || P.damage_type == BURN)
```
Comment (SomeguyManperson) [inline code/modules/antagonists/cult/cult_items.dm]: I'd say remove the burn check and maybe buff the reflect chance a bit so it still punishes laser/disabler spam to make up for the fact you can break it with WT 550s and the like

**Review by SomeguyManperson — DISMISSED** [review]

File: code/modules/antagonists/cult/cult_items.dm (hunk 1 above)
Comment (fluffe9911) [reply code/modules/antagonists/cult/cult_items.dm:999]: I was told it was a good nerf to do to prevent chain stun boomerang mirror spam

Comment (ShadowDeath6) [reply code/modules/antagonists/cult/cult_items.dm:999]: a better change would be to check if they’re paralyzed and make it not able to chain stun

**Comment by ghost** [comment]
this is terrible.

the speed robes make cultists die in two shots, you've made the cult sword inferior to common weapons to the extent that it's entirely not worth taking, and the mirror shield is now a one use joke that dies instantly in a single shot from any ranged weapon.

typical fluffe pr

File: code/modules/antagonists/cult/cult_items.dm [hunk 3]
```diff
@@ -1005,3 +1007,13 @@
 					throw_at(D.thrower, 7, 1, D.thrower)
 	else
 		..()
+
+obj/item/shield/mirror/proc/break_mirror(turf/T) //yogs mirror proc start
```
Comment (Altoids1) [inline code/modules/antagonists/cult/cult_items.dm]: Donk this proc into ``yogstation/code/modules/antagonists/cult/cult_items.dm``

**Review by Altoids1 — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/cult/cult_items.dm (hunk 3 above)
Comment (fluffe9911) [reply code/modules/antagonists/cult/cult_items.dm]: mk

**Review by Altoids1 — APPROVED** [review]

**Comment by Altoids1** [comment]
https://forums.yogstation.net/index.php?threads/vote-failed-merge-https-github-com-yogstation13-yogstation-tg-pull-5292.18995/

Council says no.

**Comment by fluffe9911** [comment]
it has been a year almost lets give this another shot

**Comment by Takahiru** [comment]
i now support this pr

**Comment by Marmio64** [comment]
i agree bloodcult needs some nerfs... but this is just nerfing a specific part way too hard. Try nerfing stunhand, and the damage nerfs to the swords is too much. Also , perhaps barrier runes should be able to be destroyed with conventional weapons but have a lot of hp?

**Comment by alexkar598** [comment]
Make this a new PR, it being at the bottom is dumb

**Pull request closed without merging**